test/index.test.js

```diff
@@ -56,7 +56,7 @@ test('reprojects', function(assert) {
     if (i > 0) assert.fail('should have only one layer');
     i++;
 
-    assert.ok(!layer.srs.isSame(sm), 'reprojected');
+    assert.ok(layer.srs.isSame(sm), 'reprojected');
```

Comment:
Hey @springmeyer this test was failing after updating node gdal. It looks like you saw that it was providing the opposite response than expected, hence the ❗. Reverting the ❗ gets tests 🍏  again but would like a 👍 👎  if this is working as expected.